docs/references/test-client.md

```diff
@@ -0,0 +1,373 @@
+    ayx-sdk-cli.exe Passthrough --input input.csv --output input.csv                        // input and output sources cannot be the same
+    ayx-sdk-cli.exe Passthrough --input input.csv:json --output.csv:json@Output2            // Output2 is non-existent
+    ayx-sdk-cli.exe Passthrough --input input.csv:csv@Input --output stdout                 // needs format
+    ayx-sdk-cli.exe Passthrough --input input.csv:csv@Input --output stdout@Output          // needs format
+
+#### Example 3
+
+These are examples of something unusual, like the Optional tool:
+
+    ayx-sdk-cli.exe Optional --input input.csv --output output.csv       // ok
+    ayx-sdk-cli.exe Optional --output output.csv                         // also ok
+    ayx-sdk-cli.exe Optional --input input.csv --output input.csv        // input/output cannot share sources
+
+#### Example 4
+
+These are examples of the test client with a Multi-Connection tool:
+
+    ayx-sdk-cli.exe MultiConnection 
+        --input input.csv --input input.csv --input input.csv --input input.csv --input input.csv 
+        --output output_one.csv@Output1 --output output_two.csv@Output2 --output output_three.csv@Output3 --output output_four.csv@Output4 --output output_five.csv@Output5     // ok!, input gets auto-mapped (multi-connection, but one anchor)
+    ayx-sdk-cli.exe MultiConnection 
+        --input input.csv --input input.csv --input input.csv --input input.csv --input input.csv 
+        --output output_one.csv@Output1 --output output_two.csv@Output2 --output output_three.csv@Output3 --output output_four.csv@Output4                                      // error! forgot Output5!
+    ayx-sdk-cli.exe MultiConnection 
+        --input input.csv --input input.csv --input input.csv --input input.csv --input input.csv 
+        --output output_one.csv@Output1 --output output_two.csv@Output2 --output output_three.csv@Output3 --output output_four.csv@Output4 --output output_five.csv@Output6     // error! no Output 6!
+
+
+## Advanced Usage
+
+Piping from stdin to stdout.
+
+    cat some_file.txt | ayx-sdk-cli.exe Passthrough --input :csv --output :csv
+    ...
+    1,2,3
+    4,5,6
+    7,8,9
+    ...
+
+Contents of **some_file.txt**:
+
+    1,2,3
+    4,5,6
+    7,8,9
+
+This doesn\'t require a schema:
+-  stdin is valid. It can be shared amongst any inputs, like a file.
+-  stdount is not valid. Similar to a file, multiple outputs can\'t write to it.
+
+### Example
+    cat some_file.txt | ayx-sdk-cli.exe Passthrough 
+        --input :csv
+        --input :csv
+        --input :csv
+        --input :csv
+        --input :csv
+        --output :csv@Output1
+        --output :csv@Output2
+        --output :csv@Output3
+        --output :csv@Output4
+        --output :csv@Output5
+
+
+
+## Command Reference 
+Go to [Test Client Command Reference](https://help.alteryx.com/developer-help/test-client-command-reference) for detailed information.
```

Comment:
Point this to the new reference on ayx-developer-sdk repo.

This should point to the corresponding markdown version